.vscode/cspell.json

```diff
@@ -197,7 +197,15 @@
     {
       "filename": "**/sdk/ai/**/*.cs",
       "words": [
-        "Ubinary"
+        "Ubinary",
+        "Ansii",
+        "Atbash",
+        "Bleu",
+        "azureai",
+        "bleu",
+        "azureai",
+        "Gleu",
+        "gleu"
       ]
```

Comment:
For next update/release - In .net repo is there only one cspell.json at the root? In Python we have those in every SDK folder... is it not the case for .net repo as well? If so in the next update perhpas move all of this to a local cspell.json that's just used for the Azure.AI.Projects folder.